### ISSUE TITLE
Welcome Screen

### ISSUE THREAD

**Issue by hagitta** [Prize4Life/Android#65]
Welcome screen:
Looks very overwhelming, and almost like the app has started with an error. I therefore suggest the following enhancements:

  --  Add the app’s logo and the Prize4Life at the top of the page.
  --  Place the text in the center of the page (middle-center), not top-center.
   -- Make the title “Welcome to the ALS Mobile Analyzer” a little larger than the rest of the body text.
--    Move the “Close” button to middle (not right side of screen)
 --   Rename “Close” button to something more friendly like “Get Started”
  --  Add app’s blue background and consider changing to the custom font type and color that the rest of the app uses.
Ari -  Pls assign to Guy after you fix programming so that a designer can take care of the looks. Thanks!

**Comment by hagitta** [comment]
There are many issues here - some of them really need the UX person, but some are easy fixes, I think (all the middle-center alignment, the enlarged title, the changes to the close button) - can you have another look?

**Comment by arisprung** [comment]
I agree that I can fix some . But it would be a waste if I fix something up and then have to redo the screen again, I suggest we first see what we want the screen to look/do then start development.

**Comment by hagitta** [comment]
Ok

**Comment by arisprung** [comment]
Fixed
